### ISSUE TITLE
Hyperparameter tuning to improve the model performance

### ISSUE THREAD

**Issue by masoudrahimi39** [masoudrahimi39/ECG-code#6]
below parameters should be adjusted:
learning rate
added layers 
loss function

MLflow for visualization of the training process
